web/templates/blocks/health_container.html.tmpl

```diff
@@ -2,16 +2,16 @@
     <h5>Health</h5>
     <div class="health-container{{ if eq .Layout "horizontal" }} horizontal-container{{ end }}">
         <div class="alert alert-inline alert-success">
-            <i class="eos-icons eos-18 alert-icon">check_circle</i>
+            <i class="eos-icons-outlined eos-18 alert-icon">check_circle</i>
```

Comment:
Is there some reason you are using the `outlined` version here?
How does it look like?
Could you upload a screenshot?

The same for the badges. I would like to see them working before approving hehe